prime-router/docs/design/proposals/0023-condition-to-code-mapping/0023-condition-to-code-mapping.md

```diff
@@ -0,0 +1,987 @@
+                            "display": "No"
+                        }
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+[Example Output](prime-reportstream/prime-router/docs/design/proposals/0023-condition-to-code-mapping/exampleoutput1.hl7):
+```text
+FHS|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|||20230330123236+0000
+BHS|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|||20230330123236+0000
+MSH|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|Simple Report^CDPH000085^CLIA|CDPH CA REDIE^2.16.840.1.114222.4.3.3.10.1.1^ISO|CDPH_CID^2.16.840.1.114222.4.1.214104^ISO|20230330123236+0000||ORU^R01^ORU_R01|1519ae86-98f4-40d8-9780-d5211186e5b5|P|2.5.1|||||USA|UNICODE UTF-8|||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
+SFT|Centers for Disease Control and Prevention|2c5a603|PRIME ReportStream|2c5a603||20230329192441+0000
+PID|1||9473889b-b2b9-45ac-a8d8-191f27132912^^^CSV uploads&11D1111111&CLIA^PI||Patient^Test^Middle^^^^L||19601001|F||2054-5^black^HL70005^^^^2.5.1|123 Main St^^Orange^CA^92868^USA||^PRN^CP^^1^555^5555555^^^^^(555) 555 55555~^NET^Internet^test.email@gmail.com|||||||||N^Not Hispanic or Latino^HL70189^^^^2.9
+ORC|RE||1519ae86-98f4-40d8-9780-d5211186e5b5^CSV uploads^11D1111111^CLIA|||||||||^Roberts^Robert^M||^WPN^PH^^1^882^8888888^^^^^(882) 888 8888|20230330053236-0700||||||CSV uploads|123 Main St^^^CA^94553^USA|^WPN^PH^^1^888^8888888^^^^^(832) 888 8888|^^^^^USA
+OBR|1||1519ae86-98f4-40d8-9780-d5211186e5b5^CSV uploads^11D1111111^CLIA|97099-6^^LN|||20230330051736-0700|||||||||^Roberts^Robert^M|^WPN^PH^^1^832^8888888^^^^^(832) 888 8888|||||20230330051736-0700|||F
+OBX|1|CE|97099-6^COVID-19^LN||260415000^Not detected^SCT|||N^Normal^HL70078^^^^2.7|||F||||||^BD Veritor System for Rapid Detection of SARS-CoV-2 \T\ Flu A+B*^^^^^^^BD Veritor System for Rapid Detection of SARS-CoV-2 \T\ Flu A+B*||||||CSV uploads-11D1111111^^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^11D1111111|123 Main St^^^CA^94553^USA
+NTE|1|L|95419-8 Has symptoms related to condition of interest: No N HL70136
+SPM|1|1519ae86-98f4-40d8-9780-d5211186e5b5&CSV uploads&11D1111111&CLIA^1519ae86-98f4-40d8-9780-d5211186e5b5&CSV uploads&11D1111111&CLIA||697989009^Anterior Nasal Swab^SCT^^^^2.67|||||||||||||20230329051736-0700|20230329051742-0700
+BTS|1
+FTS|1
+```
+During the routing step the results of the condition filter are evaluated and for observations which pass the filter a reference to that observation is added to the FHIR bundle endpoint resource. During the Translate step any observations not referenced in the endpoint resource of the FHIR bundle will not be added to the outgoing report to the receiver. This will include both observations where a mapping could not be found as well as observations representing a reportable condition that is not of interest to the receiver.  
```

Comment:
When we receive FHIR, is it possible some conditions would already be in the endpoint resource? If so, we probably need a different way to communicate if an observation is wanted or not, maybe with another variable in the RS condition extension?